### DISCUSSION TITLE
How can i return String when use @ActionMapping

### DISCUSSION THREAD

**Discussion by kyowanghwe** [dentsusoken/iPLAss#1669] [Q&A]
I saw that currently maybe we can use the DYNAMIC result type, but I don't know how to implement it. Could you provide an example?
![image](https://github.com/user-attachments/assets/810566ba-cd71-40da-84fb-52cde0bf3e20)

**Comment by KoheiKGYM** [comment]
Hi @kyowanghwe,
Thanks for using iPLAss!

Type.DYNAMIC is a type where the template to be returned is dynamically specified. Specifically, by using DYNAMIC, you can select a different template based on the process within the Command.
It is possible that this DYNAMIC may not align with the requirements you want to achieve.

Please refer to the following link for more details:
[DynamicTemplate](https://iplass.org/docs/developerguide/customizing/index.html#:~:text=%E5%8F%82%E7%85%A7%E3%81%8F%E3%81%A0%E3%81%95%E3%81%84%E3%80%82-,DynamicTemplate,-%E5%8B%95%E7%9A%84%E3%81%ABTemplate)

If you could provide more details about the specific requirements you want to achieve, I believe I can offer more suitable advice.
Thank you!
